exercises/reverse-string/example.go

```diff
@@ -1,7 +1,7 @@
 package reverse
```

Comment:
I remember this one.  See https://github.com/exercism/go/pull/971
and the discussion about the func name.
In actual client code (of which there is none for an exercise), you would have reverse.String(x). With the change, a caller would use reverse.Reverse(x). A bit wordy, but I understand the reason for the PR.
@bitfield any new thoughts on this one ?